src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -93,18 +95,31 @@ const StateUpdatedTemplate = hbs`
 
 const CommentView = View.extend({
```

Comment:
would it make sense to move this over to `comments_views` and import it here?

Reply:
I've re-written this response 4 times now because I really can't decide where it should live. I was worried this might be too specific to the sidebar to live in a shared file.

Reply:
Yeah I think this is an activity view representing a comment, and the other view is very comment related.  _maybe_ activity at some point would be extracted from the sidebar, but for the moment I'd argue it's more related to activity than the comment

Reply:
makes sense to me. That was as much me wondering aloud than anything else, so I'm happy with it staying put